Makefile

```diff
@@ -338,7 +338,8 @@ generate-clients: \
 
 .PHONY: update-deployinator-spec
 update-deployinator-spec:
-	curl -s https://deployinator-trebuchet.prod.atl-paas.net/api/swagger.json | jq '.' > pkg/releases/deployinator-trebuchet.json
+	curl -s https://contract-testing-broker.us-east-1.prod.atl-paas.net/providers/deployinator-trebuchet/spec \
+	 | jq -r .content  | base64 -D | jq -S > pkg/releases/generated/deployinator-trebuchet.json
```

Comment:
Not everyone has `base64` installed sadly :cry: 
```
python -m base64 -d
```

I'd also wrap `.content`:
```
jq -r '.content'
```

It might be good to show the errors for curl as well, they'll go to `stderr`:
```
curl -Ss ...
```